### PR TITLE
chore(test): disable browserstack testing for non-master

### PR DESCRIPTION
**Pull request by rchl**

### CODE REVIEW

**Comment by pimlie** [comment]
Thanks but unfortunately this is only a partial fix, the browserstack env's are only not added on pull requests from fork's. Eg if I would add a branch on this repo and create a pr, then it does work. But if I would create a branch on `github.com/pimlie/tib` and create a pr from that it would also not work.

Looking at the circleci docs I think the filter we need is `ignore: /^pull\//`, eg the jobs from this pr are listed as [`pull/31`](https://circleci.com/gh/nuxt/tib/2194)

File: .circleci/config.yml [hunk 1]
```diff
@@ -139,7 +139,7 @@ workflows:
       - lint:                   { requires: [setup] }
       - audit:                  { requires: [setup] }
       - test-unit:              { requires: [lint] }
-      - test-e2e-browserstack:  { requires: [lint] }
+      - test-e2e-browserstack:  { requires: [lint], filters: { branches: { only: master } } }
```
Comment (pimlie) [inline .circleci/config.yml]:
```suggestion
      - test-e2e-browserstack:  { requires: [lint], filters: { branches: { ignore: /^pull\// } } }
```

**Comment by rchl** [comment]
No, it seems to just match the branch name but seems like regexp has to match whole branch name so that's why `/pull\/.+` works.

**Comment by rchl** [comment]
Ready to merge?

**Pull request merged**